styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java

```diff
@@ -602,6 +625,15 @@ private static DockerRunner createDockerRunner(
         serviceAccountKeyManager, debug, styxEnvironment, secretWhitelist));
   }
 
+  private static FlyteRunner createFlyteRunner(Environment environment){
+    final Config config = environment.config();
+    var channel =
+        ManagedChannelBuilder.forAddress(config.getString(FLYTEADMIN_HOST),
+            config.getInt(FLYTEADMIN_PORT)).usePlaintext().build();
```

Comment:
is plaintext connect here ok, or should this be configurable?

Reply:
Oups yes, this was just copy paste. Should it ever be usePlainText?

Reply:
I will make it configurable.